src/main/resources/resources.properties

```diff
@@ -125,7 +125,12 @@ ui.authenticationView.noEmptyToken=The token cannot be empty
 ui.authenticationView.tokenLink=What is my token?
 
 ### Misc
-ui.aboutDialog.description=<html>The A+ Courses plugin supports the educational use of IntelliJ (and its Scala plugin) in<br>programming courses that rely on the A+ learning management system, which has been<br>developed at Aalto University. The plugin accesses programming assignments<br>and automated grading services provided by A+ and otherwise enhances<br>the student experience.</html>
-ui.aboutDialog.title=About the A+ Courses Plugin
+ui.aboutDialog.description=<html>The A+ Courses plugin supports the educational use of IntelliJ (and its Scala plugin)<br>in programming courses that rely on the A+ learning management system, which<br>has been developed at Aalto University.<br><br>The plugin accesses programming assignments and automated grading services<br>provided by A+ and otherwise enhances the student experience.</html>
+ui.aboutDialog.title=A+ Courses
 ui.aboutDialog.website=A+ Courses Plugin website
 ui.localHistory.submission.tag={0}, {1}, Submission {2}
+ui.aboutDialog.version=<html><b>Version: {0}</b></html>
+ui.aboutDialog.authors=<html>Developed by the department of Computer Science at Aalto University.</html>
+ui.aboutDialog.APlusWebsite=A+ website
+ui.aboutDialog.GithubWebsite=A+ Courses Plugin Github
```

Comment:
This is a nitpicky minor detail, but since this is our "very official and professional" about window, let's capitalize the 'H' in GitHub

Reply:
The existing link also had "Plugin" with a uppercase 'p', but to be consistent with the first paragraph "The A+ Courses plugin...", I suggest we change to lowercase 'p' in both of the links. (_very_ minor detail again, but I can't help myself)

Reply:
Well spotted! :smile: